_config/cache.yml

```diff
@@ -0,0 +1,5 @@
+SilverStripe\Core\Injector\Injector:
```

Comment:
Can you give this config block a name please? This allows other modules to modify it with before and after priority